packages/block-library/src/group/deprecated.js

```diff
@@ -8,7 +8,80 @@ import classnames from 'classnames';
  */
 import { InnerBlocks, getColorClassName } from '@wordpress/block-editor';
 
+const migrateCustomColors = ( attributes ) => {
+	if ( ! attributes.customTextColor && ! attributes.customBackgroundColor ) {
+		return attributes;
+	}
+	const style = { color: {} };
+	if ( attributes.customTextColor ) {
```

Comment:
What about textColor and backgroundColor? What’s the reason the aren’t mapped?

Reply:
Because the value of these attributes didn't change.

Reply:
I see the comment from @jorgefilipecosta that expands on the same in https://github.com/WordPress/gutenberg/pull/21021#discussion_r395272126. We should further explore separately whether we can get simplify the implementation to use only one attribute per color type.